packages/util/json/index.ts

```diff
@@ -0,0 +1,13 @@
+export default {
```

Comment:
I had found a library to tackle json with bigint, you can take the code below as reference.

https://github.com/pilagod/waallet-extension/blob/add-popup-authorizer/packages/util/json/index.ts

Reply:
Yes! Additionally, I found that using the same code as Cyan here (json-bigint) can resolve the only conflict issue.